parijat-frontend/src/routes/content/index.js

```diff
@@ -21,7 +22,7 @@ const Item = ({ result }) => (
 		</h4>
 		<div>
 			{result.text.split('\n').map(paragraph => (
```

Comment:
I wonder if we need to `split` and render each lines on our own if we are using `html` to render. Have you tried that? @PragyaTripathi

Reply:
I did, Prasanna. \n didn't have the same formatting. I like the layout better after putting it in <p> tag.